src/io/xpi.ts

```diff
@@ -57,7 +58,14 @@ export class Xpi extends IOBase {
 
       this.zipLib.open(
         this.path,
-        { autoClose: this.autoClose },
+        {
+          autoClose: this.autoClose,
+          // Enable checks on invalid chars in zip entries filenames.
+          strictFileNames: true,
+          // Decode automatically filenames and zip entries content from buffer into strings
+          // and autodetects their encoding.
```

Comment:
Maybe add that, without this option set to `true`, `strictFileNames` has no effect.

Reply:
@willdurand good point, I forgot to make that clear after I did split that into two separate comments